tests/ParserTest.elm

```diff
@@ -596,6 +596,14 @@ expr =
                   )
                 ]
               )
+            , ( "unit is unit"
+              , [ ( "unit"
```

Comment:
Let's keep the outer description for the expression case (in this case `"unit"`) and the inner one for what is the specific example testing. In this case, there's not much to test, so I'd just write `"simple case"`.